diff --git a/node_modules/@sap/cds/libx/rest/RestAdapter.js b/node_modules/@sap/cds/libx/rest/RestAdapter.js
index e987da7..34540f1 100644
--- a/node_modules/@sap/cds/libx/rest/RestAdapter.js
+++ b/node_modules/@sap/cds/libx/rest/RestAdapter.js
@@ -122,7 +122,11 @@ const RestAdapter = function (srv) {
     }
     next()
   })
-  router.use(express.json()) // REVISIT: -> belongs to the parses
+
+  
+  console.log('MST - REST Adapter Patch: Set limit to 50mb')
+  router.use(express.json({limit:'50mb'})) // REVISIT: -> belongs to the parses
+  //router.use(express.json()) // REVISIT: -> belongs to the parses
   router.use(parse) // REVISIT: -> move to actual handler(s)
   router.use(payload) // REVISIT: -> move?
   // payload validation
